firebase-firestore/src/androidMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -66,6 +72,8 @@ actual class FirebaseFirestore(val android: com.google.firebase.firestore.Fireba
 
 actual class WriteBatch(val android: com.google.firebase.firestore.WriteBatch) {
 
+    actual val async = Async(android)
```

Comment:
in order not to break existing API each class that supports async operations provides access to an `async` field